plugins/contrast/index.js

```diff
@@ -6,6 +6,7 @@
 let $ = require("jquery");
 let Plugin = require("../base");
 let annotate = require("../shared/annotate")("labels");
+let blinder = require("../../node_modules/color-blind/lib/blind").Blind;
```

Comment:
Oh, you're absolutely right! I was looking at the "main" field in package.json, but that points to `lib/color-blind`. Anyway - thanks.